linux-build.proj

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0"
+         DefaultTargets="Build"
+         xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <RHostName Condition=" '$(RHostName)'=='' ">Microsoft.R.Host</RHostName>
+    <RootDirectory Condition=" '$(RootDirectory)'=='' ">$(MSBuildThisFileDirectory)..\..\..\</RootDirectory>
+    <OutputDirectory Condition=" '$(OutputDirectory)'=='' ">$(RootDirectory)bin\</OutputDirectory>
+    <IntermediateOutputDirectory Condition=" '$(IntermediateOutputDirectory)'=='' ">$(RootDirectory)obj\$(RHostName)\</IntermediateOutputDirectory>
+    <SourceDirectory Condition=" '$(SourceDirectory)'=='' ">$(MSBuildThisFileDirectory)src\</SourceDirectory>
+    
+
+    <Year Condition="'$(Year)' == ''">$([System.DateTime]::Now.Year)</Year>
+    <Date Condition="'$(Date)' == ''">$([System.DateTime]::Now.ToString("MMdd"))</Date>
+    <Time Condition="'$(Time)' == ''">$([System.DateTime]::Now.ToString("HHmm"))</Time>
+
+    <RHostDirectory Condition="'$(RHostDirectory)' == ''">$(MSBuildThisFileDirectory)</RHostDirectory>
+    <RHostSH Condition="'$(RHostSH)' == ''">$(RHostDirectory)build.sh</RHostSH>
+    <SolutionProperties Condition="'$(SolutionProperties)' == ''">
+      Configuration=$(Configuration);
+      Platform=$(Platform);
+      GenerateAssemblyVersionInfo=true;
+      Year=$(Year);Date=$(Date);Time=$(Time);
+      SolutionDir=$(SourceDirectory);
+      BinDirectory=$(OutputDirectory);
+      ObjDirectory=$(IntermediateOutputDirectory);
+      LibDirectory=$(LibDirectory);
+    </SolutionProperties>
+  </PropertyGroup>
+
+  <Target Name="Restore">
+  </Target> 
+
+  <Target Name="Clean">
+    <Delete Files="$(OutputDirectory)$(RHostName)" />
+  </Target>
+
+  <Target Name="Rebuild">
```

Comment:
This is basically clean + rebuild, or it should be - so it can be implemented as `DependsOnTargets="Clean;Rebuild"` (and with empty body).